src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -207,6 +188,68 @@ public String getScript() {
         return c;
     }
 
+    private String binaryLauncherCmd(ShellController c, FilePath ws, OsType os, String interpreter, String scriptPath, String cookieValue, String cookieVariable) throws IOException, InterruptedException {
+        String logFile = c.getLogFile(ws).toURI().getRawPath();
+        String resultFile = c.getResultFile(ws).toURI().getRawPath();
+        FilePath controlDir = c.controlDir(ws);
+        String outputFile = c.getOutputFile(ws).toURI().getRawPath();
+
+        String launcher_binary = LAUNCHER_PREFIX + os.toString();
+
+        InputStream launcherStream = this.getClass().getResourceAsStream(launcher_binary);
```

Comment:
Tests failing with

```
java.lang.NullPointerException
	at org.apache.commons.io.IOUtils.copyLarge(IOUtils.java:1792)
	at org.apache.commons.io.IOUtils.copyLarge(IOUtils.java:1769)
	at org.apache.commons.io.IOUtils.copy(IOUtils.java:1744)
	at hudson.FilePath.copyFrom(FilePath.java:913)
	at org.jenkinsci.plugins.durabletask.BourneShellScript.binaryLauncherCmd(BourneShellScript.java:201)
	at org.jenkinsci.plugins.durabletask.BourneShellScript.launchWithCookie(BourneShellScript.java:161)
	at org.jenkinsci.plugins.durabletask.FileMonitoringTask.launch(FileMonitoringTask.java:99)
	at org.jenkinsci.plugins.durabletask.BourneShellScriptTest.smokeTest(BourneShellScriptTest.java:99)
```

```suggestion
        InputStream launcherStream = this.getClass().getResourceAsStream(launcher_binary);
        if (launcherStream == null) {
            throw new IOException("Could not find a launcher for this OS: " + launcher_binary);
        }
```

Though better would be to fall back to a simplified version of the original shell script wrapper, as requested in https://github.com/jenkinsci/durable-task-plugin/pull/92#discussion_r282082078, meaning we do this null check instead of the current

```java
if ((os == OsType.DARWIN) || (os == OsType.UNIX)) {
```